R/getDominantTaxa.R

```diff
@@ -0,0 +1,152 @@
+#' Get Dominant Taxa
+#'
+#' This function return the most dominant taxa.
+#'
+#' @param x
+#' A \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#' object
+#'
+#' @param rank A single character defining a taxonomic rank. Must be a value of
+#'   \code{taxonomicRanks()} function.
+#'
+#' @param group With group, it is possible group observations in overview.
+#'
+#' @param name A name for the column of the colData where the dominant taxa
+#' will be stored in.
+#'
+#' @details
+#' \code{getDominantTaxa} extracts the most abundant taxa
+#' in a \code{\link[=SummarizedExperiment-class]{SummarizedExperiment}} object.
+#'
+#' @return \code{x} with additional \code{\link{colData}} named
+#'   \code{*name*}
+#'
+#' @name getDominantTaxa
+#' @export
+#'
+#' @author Leo Lahti and Tuomas Borman. Contact: \url{microbiome.github.io}
+#'
+#' @examples
+#' data(GlobalPatterns)
+#' x <- GlobalPatterns
+#'
+#' #Finds the dominant taxa.
```

Comment:
I would prefer commenting where you have a space after # i.e. "# Finds the dominant taxa". Can this be changed here and elsewhere?

Reply:
Added space after # in estimateDominance and getDominantTaxa